internal/db/db.go

```diff
@@ -43,6 +43,13 @@ var (
 	_ client.Collection = (*collection)(nil)
 )
 
+const (
+	// commandBufferSize is the size of the channel buffer used to handle events.
+	commandBufferSize = 100
```

Comment:
todo: I think the default should be a much larger value, maybe 100,000 for now. 100,000 would still only be 1.6MB I think, and stuff will start blocking if the capacity is exceeded.